blocks/context/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External depencencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * This is a stopgap until React 16.3 is released with the new context API.
+ *
+ * @param {Object} defaults The context defaults.
+ *
+ * @return {Object} an object with a context Provider and Consumer.
+ */
+export function createContext( defaults ) {
+	const contextTypes = Object.keys( defaults ).reduce(
+		( types, key ) => ( types[ key ] = noop, types ),
+		{}
+	);
+
+	class Provider extends Component {
+		getChildContext() {
+			return {
+				...defaults,
+				...this.props.value,
+			};
+		}
+
+		render() {
+			return (
+				<Fragment>
```

Comment:
You shouldn't need a `<Fragment>` here, just:

```js
return this.props.children;
```

Reply:
Good to know. Thanks!